build/npm/navigation-react-native/navigation.react.native.d.ts

```diff
@@ -78,6 +78,11 @@ export class LeftBarIOS extends Component { }
  */
 export class RightBarIOS extends Component { }
 
+/**
+ * Renders titleView in the UI navigation bar
+ */
+export class TitleBarIOS extends Component {  }
```

Comment:
The `TitleBarIOS` definitions need a [style prop](https://github.com/grahammendick/navigation/blob/753ecc6ad8c3735d583ccd2baa76a12239a5e19e/NavigationReactNative/test/node_modules/%40types/navigation-react-native.d.ts#L165).

You can test that your typescript definitions work by adding your component to [navigation-react-native-test.tsx](https://github.com/grahammendick/navigation/blob/master/NavigationReactNative/test/navigation-react-native-tests.tsx). Think of the file as a unit test for the typings. Then `cd` into the 'test' folder and run `tsc --jsx react --target es6 navigation-react-native-tests.tsx`. Check the js file is generated without any errors or warnings

Reply:
Oh, didn't notice this test file.

Reply:
You don't have to commit it, but you can use it to test your change.